modules/rostests/kmtests/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ list(APPEND KMTEST_DRV_SOURCE
     kmtest_drv/guid.c
     kmtest_drv/kmtest_drv.c
     kmtest_drv/testlist.c
+    kmtest_drv/structs.c
```

Comment:
We usually sort the files in alphabetical order (should be above testlist.c).

Reply:
Will fix before pushing.